backend/Tasque.BLL/Services/TaskStateService.cs

```diff
@@ -16,7 +16,7 @@ public TaskStateService(DataContext db, IMapper mapper, CurrentUserParameters cu
         
         }
 
-        public new async Task<IEnumerable<TaskState>> GetAll()
+        public async Task<IEnumerable<TaskState>> GetAll()
         {
             var tasksStates = await _db.TaskStates
```

Comment:
Remove `async` from the method's signature and just do `return _db.TaskStates.ToListAsync()`